src/radical/entk/task.py

```diff
@@ -1,25 +1,110 @@
-__copyright__ = 'Copyright 2014-2020, http://radical.rutgers.edu'
+# pylint: disable=protected-access
+
+__copyright__ = 'Copyright 2014-2022, The RADICAL-Cybertools Team'
 __license__   = 'MIT'
 
 import warnings
 
+from string import punctuation
+
 import radical.utils as ru
 
-from string     import punctuation
 from .constants import NAME_MESSAGE
-
 from . import exceptions as ree
 from . import states     as res
 
-warnings.simplefilter(action="once", category=DeprecationWarning, lineno=728)
```

Comment:
off topic: we should at some point discuss when we will retire the deprecation warnings.